.grabl/automation.yml

```diff
@@ -28,19 +28,16 @@ build:
       command: |
         bazel build //...
         bazel run @vaticle_dependencies//tool/checkstyle:test-coverage
-        bazel test $(bazel query 'kind(checkstyle_test, //...)') --test_output=streamed
+        bazel test $(bazel query 'kind(checkstyle_test, //...)')
     test-unit:
       image: vaticle-ubuntu-20.04
       command: |
         bazel test $(bazel query 'kind(kt_jvm_test, //...) except kind(kt_jvm_test, //test/...)')
     test-integration:
       image: vaticle-ubuntu-20.04
       command: |
-        bazel run //:typedb-extractor -- dist/typedb-all-linux
-        sudo systemd-run ./dist/typedb-all-linux/typedb server
-        sleep 60
         bazel build //test/integration/...
-        bazel test $(bazel query 'kind(kt_jvm_test, //test/integration/...)') --test_output=streamed --spawn_strategy=local
+        bazel test $(bazel query 'kind(kt_jvm_test, //test/integration/...)') --jobs=1 --spawn_strategy=local
```

Comment:
Agreed, and as such we're explicit about the fact we just want one job to be running at any given time rather than as a side effect of --test_output=streamed.